packages/nimble-components/src/wafer-map/modules/prerendering.ts

```diff
@@ -51,9 +51,11 @@ export class Prerendering {
         const isDieRenderInfo = (
             info: DieRenderInfo | null
         ): info is DieRenderInfo => info !== null;
-        this._diesRenderInfo = this.wafermap.dies
-            .map(die => this.computeDieRenderInfo(die))
-            .filter(isDieRenderInfo);
+        if (this.wafermap.diesTable === undefined) {
```

Comment:
maybe you can use teh isExperimental() check here

Reply:
yes I can, did it

Reply:
was that change made? I don't see it here

Reply:
It was made but at that time I removed it because it was causing some errors in the storybook, I don't remember in detail. I will solve this conversation, the explanation is given in another conversation.